pkgs/development/ocaml-modules/stdint/default.nix

```
@@ -2,15 +2,15 @@
 
 buildDunePackage rec {
   pname = "stdint";
-  version = "0.6.0";
+  version = "0.7.0";
 
   minimumOCamlVersion = "4.07";
```

Introduces support with lower OCaml versions.

```suggestion
  minimumOCamlVersion = "4.03";
```